### ISSUE TITLE
App_Config / Include configuration files does not have support for Solr

### ISSUE THREAD

**Issue by davidnguyen** [Sitecore/Sitecore-Configuration-Roles#9]
Hi, I was looking to use Sitecore-Configuration-Roles but realised that the re-baseline App_Config / Include files does not have support for Solr, all the index related .config files are Lucene related. Is there any plan to support Solr in the future, or any idea how to get around it at the moment?

**Comment by AlenPelin** [comment]
@davidnguyen yes, Solr is out of scope for now. It will be added later this or next month.

**Comment by ewerkman** [comment]
Any update on this?

**Comment by AlenPelin** [comment]
Unfortunately no, priorities changed.
